apps/site/components/Downloads/DownloadReleasesTable/index.tsx

```diff
@@ -21,35 +19,27 @@ const DownloadReleasesTable: FC = async () => {
           <th>{t('components.downloadReleasesTable.version')}</th>
           <th>{t('components.downloadReleasesTable.nApiVersion')}</th>
           <th>{t('components.downloadReleasesTable.codename')}</th>
-          <th>{t('components.downloadReleasesTable.releaseDate')}</th>
-          <th>{t('components.downloadReleasesTable.npmVersion')}</th>
+          <th>{t('components.downloadReleasesTable.firstReleased')}</th>
+          <th>{t('components.downloadReleasesTable.lastUpdated')}</th>
+          <th>{t('components.downloadReleasesTable.status')}</th>
           <th></th>
         </tr>
       </thead>
       <tbody>
         {releaseData.map(release => (
           <tr key={release.major}>
-            <td data-label="Version">v{release.version}</td>
+            <td data-label="Version">v{release.major}</td>
             <td data-label="Modules">v{release.modules}</td>
             <td data-label="LTS">{release.codename || '-'}</td>
+            <td data-label="Date">
+              <time>{release.currentStart}</time>
+            </td>
             <td data-label="Date">
               <time>{release.releaseDate}</time>
             </td>
-            <td data-label="npm">v{release.npm}</td>
+            <td data-label="Status">{release.status}</td>
```

Comment:
Wouldn't it be interesting if the value was in the form of a “pils” with the color assigned to it in the cart above?